linux_os/guide/system/bootloader-zipl/zipl_audit_argument/tests/correct_option.pass.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# platform = Red Hat Enterprise Linux 8
```

Comment:
I wouldn't be so strict with the platform - it is a textfilecontent test, so actually any platform will do. The remediation requires `grubby`, but that can be installed into Fedora as well.

Reply:
I enabled it for Fedora as well, and removed `# remediation = none`.